src/client.zig

```diff
@@ -239,7 +239,7 @@ pub const Client = struct {
         self.message_bus.unref(queued_request.message);
 
         if (self.request_queue.peek()) |next_request| {
-            self.send_request(next_request.message);
+            self.send_request(next_request.*.message);
```

Comment:
This isn't necessary to access a field of the pointed to struct, the semantics are exactly the same as the status quo code.
```suggestion
            self.send_request(next_request.message);
```